### DISCUSSION TITLE
AIR 51.0.1.2 error while loading initial content ERROR dissolvent

### DISCUSSION THREAD

**Discussion by semoki** [airsdk/Adobe-Runtime-Support#3279] [Guides & Tutorials]
After downloading AIR 51.0.1.2, an "error while loading initial content" error occurred when building in Animate CC. In some ways, it is the most basic method, but I am uploading it because I think it will be helpful for beginners like me. You must change ns.adobe.com/air/sdk/50.2" to " airSdk xmlns="http://ns.adobe.com/air/sdk/51.0" to avoid errors. This is a very basic part. It's a beginner part, but I'm lost because I don't know. I'm writing this in the hope that it will be helpful to beginners like me.

**Comment by zanorg** [comment]
Thank you !
